osf/utils/fields.py

```diff
@@ -95,7 +99,7 @@ def get_prep_value(self, value, **kwargs):
 
     def to_python(self, value):
         value = rapply(value, decrypt_string, prefix=self.prefix)
-        return super(EncryptedJSONField, self).to_python(value)
+        return super(EncryptedJSONField, self).from_db_value(value, None, None)
 
-    def from_db_value(self, value, expression, connection, context):
+    def from_db_value(self, value, expression, connection):
         return self.to_python(value)
```

Comment:
Similar but slightly complicated case. Ideally, we only need to do the `from_db_value()` but since `to_python()` was defined/used before, we keep the egress logic intact but update the internal one to use  the super `from_db_value()`.